### DISCUSSION TITLE
What's the best way to restore my ZigBee network in my case ?

### DISCUSSION THREAD

**Discussion by jlpouffier** [Koenkk/zigbee2mqtt#17133] [General]
Hey 👋

I run a small french YouTube channel on Home Assistant and plan to make a video on installing and configuring Zigbee2MQTT.
But I don't really have spare hardware to do that on a test network 

For now, my plan is to

- Back-up my production Zigbee network (With what Z2M offers under Settings > Tools) 
- Remove everything 
- Record my video so I can show how to install the add-on, configure it, and pair a few devices
- ??

Then I am pretty lost... How do I restore everything with the backup file I extract on step 1?

Thx a lot :)

**Comment by junosuarez** [comment]
That video sounds cool! A similar question was asked recently, see https://github.com/Koenkk/zigbee2mqtt/discussions/17085#discussioncomment-5397353

**Reply by jlpouffier** [reply]
Very clear. Thx a lot !
